saos-enrichment/src/main/java/pl/edu/icm/saos/enrichment/delete/JudgmentWithEnrichmentDeleter.java

```diff
@@ -0,0 +1,72 @@
+package pl.edu.icm.saos.enrichment.delete;
+
+import java.util.List;
+
+import javax.persistence.EntityManager;
+import javax.persistence.Query;
+import javax.transaction.Transactional;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Service;
+
+import com.google.common.collect.Lists;
+
+import pl.edu.icm.saos.enrichment.reference.JudgmentReferenceRemover;
+import pl.edu.icm.saos.persistence.enrichment.EnrichmentTagRepository;
+import pl.edu.icm.saos.persistence.enrichment.model.EnrichmentTag;
+import pl.edu.icm.saos.persistence.model.Judgment;
+import pl.edu.icm.saos.persistence.repository.JudgmentRepository;
+
+/**
+ * @author madryk
+ */
+@Service
+public class JudgmentWithEnrichmentDeleter {
+
+    @Autowired
+    private EntityManager entityManager;
+    
+    @Autowired
+    private JudgmentRepository judgmentRepository;
+    
+    @Autowired
+    private EnrichmentTagRepository enrichmentTagRepository;
+    
+    @Autowired
+    @Qualifier("judgmentReferenceRemover")
+    private JudgmentReferenceRemover judgmentReferenceRemover;
```

Comment:
TagJudgmentReferenceRemover? Because now one can not know what reference it is about.